practice-two/src/css/main.css

```diff
@@ -0,0 +1,79 @@
+.container-main {
+  padding: 0 16px;
+}
+.nav-item {
+  display: flex;
+  justify-content: space-between;
+  padding: 32px 0;
+}
+.logo-item {
+  display: flex;
+  .logo {
+    width: 40px;
+    height: 40px;
+  }
+  .logo-text {
+    color: #191825;
+    font-size: 24px;
+    font-style: normal;
+    font-weight: 900;
+    line-height: 40px;
+    padding-left: 16px;
+  }
+}
+.menu-icon {
+  width: 48px;
+  height: 48px;
+}
+.nav-list,
+.header-button-group {
+  display: none;
+}
+@media screen and (min-width: 1024px) {
```

Comment:
Will the mobile UI work as expected If we don't use this @media here?

Reply:
it wil still working, @media here is responsive for tablet